Website/src/rendering/tsconfig.json

```diff
@@ -2,10 +2,18 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "components/*": ["src/components/*"],
-      "lib/*": ["src/lib/*"],
-      "temp/*": ["src/temp/*"],
-      "assets/*": ["src/assets/*"]
+      "components/*": [
```

Comment:
Next.js now adds `intremental: true` when running `npm run build`. This is fine. However it also reformats this file at the same time.

We got this file from the JSS starter kit and we want to keep it as close to the original as possible to facilitate future upgrades.

Please revert formatting changes to this file.